client/debian/tests/run-hwctl-tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+set -xeu
+
+python3 debian/tests/mock_server.py &
+SERVER_PID=$!
+sleep 1  # Give server time to start
```

Comment:
The server should be ready when it signals readiness to systemd. This will race on slower hardware or virtual machines.